packages/ui/src/views/project/create-project-page.tsx

```diff
@@ -157,11 +170,24 @@ export const CreateProjectPage: FC<CreateProjectPageProps> = props => {
             />
           </Fieldset>
 
-          <Button className="mt-3 w-full" rounded type="submit" loading={isLoading} disabled={hasError}>
-            {isLoading
-              ? t('views:createProject.create.projectCreation', 'Creating project...')
-              : t('views:createProject.create.createProject', 'Create project')}
-          </Button>
+          <ControlGroup type="button">
+            <Button className="mt-3 w-full" rounded type="submit" loading={isLoading} disabled={hasError} size="lg">
+              {isLoading
+                ? t('views:createProject.create.projectCreation', 'Creating project...')
+                : t('views:createProject.create.createProject', 'Create project')}
+            </Button>
+
+            <div className="mt-3 flex items-center justify-center gap-2">
+              <div className="w-[145px] shrink border-t border-cn-borders-3" />
+              <span className="text-sm text-cn-foreground-3">{t('views:createProject.or', 'or')}</span>
+              <div className="w-[145px] shrink border-t border-cn-borders-3" />
+            </div>
+
+            {/* TODO: Update the variant of this button to outline once the component supports this style. */}
+            <Button asChild className="mt-3 w-full" rounded type="button" variant="surface" size="lg">
```

Comment:
Does `type="button"` do anything here since it's then being applied to a `Link`, which renders as an anchor tag?